unidist/core/backends/mpi/core/executor.py

```diff
@@ -6,12 +6,14 @@
 
 import weakref
 import itertools
+import time
```

Comment:
Let's use `timeit` that is more accurate.

Reply:
```timeit.default_timer``` is the same as ```time.perf_counter``` since python 3.3 - [link](https://docs.python.org/3/library/timeit.html#timeit.default_timer)

Reply:
> it disables the garbage collector to prevent that process from skewing the results by scheduling a collection run at an inopportune moment.
This makes it unusable in our case. Also an answer is 8 years old.